### ISSUE TITLE
is this still in maintenance?

### ISSUE THREAD

**Issue by muava12** [sayedulsayem/custom-css-for-elementor#9]
Hi, i like this plugin, is this still in your attention? since it has updatae 9 month ago

**Comment by sayedulsayem** [comment]
@muava12 
Yes. This is still in my attention. I am working with the breakpoints hook. I hope I will be able to release it very soon. 
Thanks
